csm_web/frontend/static/frontend/data/enrollment_policies.tsx

```diff
@@ -0,0 +1,294 @@
+          "Your mentor will be your primary contact for questions regarding your section."
+      }
+    },
+    affinitySections: {
+      title: "Affinity Sections",
+      body: {
+        p1:
+          "We offer several sections geared towards supporting students from " +
+          "backgrounds underrepresented or under-resourced in EECS and STEM. " +
+          "These include:",
+        i1_b: "Affinity Sections",
+        i1_2:
+          " - These are sections designed for students who affiliate with " +
+          "identities and communities underrepresented in the Berkeley EECS " +
+          "community. Types of affinity sections that we have offered " +
+          "previously, or hope to offer, include:",
+        i1_i1: "Woman affinity sections",
+        i1_i2: "LGBTQ+ affinity sections",
+        i1_i3: "Hispanic/Latinx affinity sections",
+        i1_i4: "Transfer sections",
+        i2_b: "Limited Experience Sections (CS 61A only)",
+        i2_2:
+          " - Limited experience sections are specifically geared towards " +
+          "students with no or limited prior experience in computer science. " +
+          "They meet twice a week rather than once a week, and do not cover " +
+          "any more material than a regular section, although they will cover " +
+          "the worksheets at a slower pace and with more conceptual discussion.",
+        i3_b: "Program-affiliated sections",
+        i3_2:
+          " - We partner with other programs on campus (e.g. EOP and SEED " +
+          "Scholars) to offer sections designated for students enrolled in " +
+          "these programs. Please only enroll in one of these sections if " +
+          "you are also enrolled in the corresponding program",
+        p2:
+          "Affinity section offerings will vary by course, as they depend on " +
+          "the availability of mentors interested in teaching these sections. " +
+          "These sections will be clearly marked on the Scheduler website when " +
+          "you enroll."
+      }
+    },
+    csmForUnit: {
+      title: "Taking CSM for a unit",
+      body: {
+        p1_1: "Students enrolled in a CSM section may ",
+        p1_b: "optionally",
+        p1_2:
+          " enroll for PNP units on CalCentral. Students are eligible for " +
+          "one unit of PNP credit per CSM section enrolled in—for example, " +
+          "if you're enrolled in CSM sections for both CS 61B and CS 70, " +
+          "you may enroll for up to two units.",
+        p2_1: "Enrollment in the unit on CalCentral is ",
+        p2_b: "not automatic",
+        p2_2:
+          " for students who enroll in a section on Scheduler. If you wish " +
+          "to enroll in the unit, you should first enroll in a section on " +
+          "scheduler, then go to CalCentral and find and add the appropriate " +
+          "number of units with this course information:",
+        li1_1: "Course number: CS 197-215",
+        li1_2: "CCN: 16190",
```

Comment:
Course number and CCN should be left out here or extracted to be more prominent.

Reply:
time to implement dynamic values and feature flags 😈 uhh i kinda have some ideas of how to do this but can share them on monday

Reply:
cnn and course number fully removed 😳